BackEnd/stripeApi.js

```diff
@@ -0,0 +1,407 @@
+//const stripe = require('stripe')('sk_test_4eC39HqLyjWDarjtT1zdp7dc');
```

Comment:
This is still unresolved. We need this require statement but the api key should be passed via the config. See my previous comment